why-promptlayer/shared-workspaces.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Shared Workspaces"
+icon: "handshake-simple"
+---
+
+# Overview
+
+The Shared Workspaces feature in PromptLayer is a collaborative tool designed to enhance teamwork and efficiency. It allows users to organize and share various data such as requests, prompt templates, and evaluations, in a central workspace.
```

Comment:
Add a sentence about collaborating with your team. Admins can add multiple teams throughout their org and view them all. This is the main valueprop, not individual organization